app/models/purchase.rb

```diff
@@ -69,6 +69,10 @@ def purchased_from_view
     vendor.nil? ? purchased_from : vendor.business_name
   end
 
+  def comment_view
+    comment.nil? ? "" : comment
+  end
```

Comment:
I don't think this deserves a full method of its own. You can replace with `purchase.comment || ''`.

Reply:
Made the changes here [803a9dd](https://github.com/rubyforgood/human-essentials/pull/4642/commits/803a9ddaa3e4637170f652b6bd7ee826cd7505f5)